db-engine/src/main/java/com/splicemachine/db/impl/sql/conn/GenericLanguageConnectionContext.java

```diff
@@ -398,6 +399,12 @@ public GenericLanguageConnectionContext(
             this.sessionProperties.setProperty(SessionProperties.PROPERTYNAME.SKIPSTATS, new Boolean(skipStats).toString());
         if (defaultSelectivityFactor > 0)
             this.sessionProperties.setProperty(SessionProperties.PROPERTYNAME.DEFAULTSELECTIVITYFACTOR, new Double(defaultSelectivityFactor).toString());
+        if (connectionProperties != null) {
+            String olapQueue = connectionProperties.getProperty("olapQueue");
```

Comment:
Aren't `SessionProperties.PROPERTYNAME.OLAPQUEUE` and `olapQueue` the same? If not, could we introduce another constant for this hardcoded `olapQueue` name?